tools/mtouch/Assembly.cs

```diff
@@ -94,6 +94,26 @@ public void CopyMdbToDirectory (string directory)
 				Application.UpdateFile (mdb_src, mdb_target);
 			}
 		}
+		
+		public void CopyMSymToDirectory (string directory)
+		{
+			string msym_src = FullPath + ".aotid.msym";
+			var dirInfo = new DirectoryInfo (msym_src);
+			if (!dirInfo.Exists) // got no aot data
+				return;
+			var subdirs = dirInfo.GetDirectories();
+			foreach (var subdir in subdirs) {
+				var destPath = Path.Combine (directory, subdir.Name.ToUpperInvariant ());
+				var destInfo = new DirectoryInfo (destPath);
+				if (!destInfo.Exists)
+					Directory.CreateDirectory (destPath);
+				var files = subdir.GetFiles ();
+				foreach (FileInfo file in files) {
+					string temppath = Path.Combine (destPath, file.Name);
+					file.CopyTo(temppath, true);
```

Comment:
Use `Driver.UpdateFile` instead, which will check time stamps and not do anything if nothing has changed.